src/webview/markdownEngine.ts

```diff
@@ -7,39 +7,70 @@ import * as os from "os";
 import * as path from "path";
 import * as vscode from "vscode";
 import { leetCodeChannel } from "../leetCodeChannel";
+import { isWindows } from "../utils/osUtils";
 
-class MarkdownEngine {
+class MarkdownEngine implements vscode.Disposable {
 
-    private readonly engine: MarkdownIt;
-    private readonly extRoot: string; // root path of vscode built-in markdown extension
+    private engine: MarkdownIt;
+    private config: MarkdownConfiguration;
+    private listener: vscode.Disposable;
 
     public constructor() {
-        this.engine = this.initEngine();
-        this.extRoot = path.join(vscode.env.appRoot, "extensions", "markdown-language-features");
+        this.reload();
+        this.listener = vscode.workspace.onDidChangeConfiguration((event: vscode.ConfigurationChangeEvent) => {
+            if (event.affectsConfiguration("markdown")) {
+                this.reload();
```

Comment:
`reload()` will refresh the configuration the markdown engine. I think we can refresh the webview as well.